panel/layout.py

```diff
@@ -118,8 +138,7 @@ def _get_model(self, doc, root=None, parent=None, comm=None):
         objects = self._get_objects(model, [], doc, root, comm)
 
         # HACK ALERT: Insert Spacer if last item in Column has no height
-        if (isinstance(self, Column) and objects and not isinstance(objects[-1], (BkWidgetBox, BkBox))
-            and getattr(objects[-1], 'height', False) is None):
+        if (isinstance(self, Column) and objects and no_height(objects[-1])):
             objects.append(BkSpacer(height=50))
```

Comment:
Add a parameter `default_height`?  Could be useful e.g. for the report mode dashboards...

Reply:
Where would the parameter live? I'd be happy to add it but maybe in a separate PR.

Reply:
Well, height=50 is used here on Panel, so presumably on Panel, so that it can be height=self.default_height

Reply:
It's specifically behind an isinstance check for Column to avoid having to duplicate the whole method there.

Reply:
That said I could add an equivalent check for Row:

``(isinstance(self, Row) and objects and not any(has_height(o) for o in objects))``

Reply:
Merging this, but I can follow up on this tomorrow.